scripts/common.js

```diff
@@ -0,0 +1,31 @@
+const lerna = require('../lerna.json')
+const { execSync } = require('child_process')
+
+module.exports = {
+  run: function run (command) {
+    return execSync(command).toString().trim()
+  },
+  getCommitId: function getCommitId () {
+    return this.run('git rev-parse --short HEAD')
+  },
+  determineVersion: function determineVersion () {
+    // Form version string - branch name and commit id
+    const commitId = this.getCommitId()
+    let branchName
+    if (process.env.BRANCH_NAME) {
+      console.log('Using BRANCH_NAME from the environment')
+      branchName = process.env.BRANCH_NAME
+    } else {
+      console.log('Using local Git repo to determine branch name (environment BRANCH_NAME may also be used)')
+      branchName = this.run('git rev-parse --abbrev-ref HEAD').replace(/\//g, '-')
+    }
+
+    // Distinguish from local use
+    const ciIndicator = (process.env.BUILDKITE ? 'ci-' : '')
```

Comment:
The `BUILDKITE` environment variable will need to be added into the `docker-compose.yml` in order for it to be passed through to the container.

Reply:
Nice spot - I added that at the last minute and assumed too much.